content/doc/data-sources.dmark

```diff
@@ -118,7 +118,7 @@ title: "Data sources"
 
         item = new_item(
           content,
-          title: 'Release notes',
+          {title: 'Release notes'},
```

Comment:
The `}` is a special character in [D*Mark](https://ddfreyne.github.io/d-mark/), and needs to be escaped with a `%`:

```suggestion
          { title: 'Release notes' %},
```

Reply:
(Also, typical Ruby style means adding spaces inside `{` and `}` — but not inside `[` and `]`.)